airflow_provider_azure_machinelearning/operators/machine_learning/endpoint.py

```diff
@@ -201,3 +201,84 @@ def execute(self, context: Context) -> None:
             raise KeyError(f"Failed to deploy endpoint {self.deployment.endpoint_name}.")
 
         return self.deployment.endpoint_name
+
+
+class AzureMachineLearningInvokeEndpointOperator(BaseOperator):
+    """
+    Executes Azure ML SDK to invoke an endpoint.
+
+    :param conn_id: The connection identifier for connecting to Azure Machine Learning.
+    :param endpoint: The spec of Azure ML endpoint
+    :waiting: When true, the operator blocks till the Azure ML endpoint deployment is completed.
+
+    Returns name(id) of the Azure ML endpoint.
+    """
+
+    ui_color = "white"
+    ui_fgcolor = "blue"
+
+    def __init__(
+        self,
+        *,
+        endpoint_name: str,
+        endpoint_type: str,
+        inputs: dict = {},
+        waiting: bool = False,
+        conn_id: str = None,
+        **kwargs,
+    ) -> None:
+
+        super().__init__(**kwargs)
+
+        self.endpoint_name = endpoint_name
+        self.endpoint_type = endpoint_type
+        self.inputs = inputs
+        self.waiting = waiting
+        self.conn_id = conn_id
+        self.hook = None
+        self.ml_client = None
+        self.job = None
+
+    def execute(self, context: Context) -> None:
+
+        self.log.info(
+            f"Executing { __class__.__name__} to deploy Endpoint {self.endpoint_name}."
+        )
+        self.hook = AzureMachineLearningHook(self.conn_id)
+        self.ml_client = self.hook.get_client()
+        
+        try:
+            self.get_caller().get(name=self.endpoint_name)
+        except AmlExceptions.ResourceNotFoundError:
+            self.log.info(f"Endpoint {self.endpoint_name} does not exist. Returning.")
+            return self.endpoint_name
+
+        self.log.info(f"Invoking Endpoint: {self.endpoint_name}.")
+        self.job = self.get_caller().invoke(endpoint_name=self.endpoint_name, inputs=self.inputs)
```

Comment:
Maybe it is a good idea to wrap a try-except block on this invoke call cause it goes over the network, etc

Reply:
I did change will be published in next commit. I intercept: ServiceRequestError error